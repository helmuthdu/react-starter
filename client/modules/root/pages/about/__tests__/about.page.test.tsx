import { render } from '@testing-library/react';
import React from 'react';
import AboutPage from '../about.page';

describe('home/page -> About', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<AboutPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
