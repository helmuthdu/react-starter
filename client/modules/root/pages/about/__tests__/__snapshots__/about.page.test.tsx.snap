// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home/page -> About should match snapshot 1`] = `
<DocumentFragment>
  <section>
    <p>
      This project is created by @helmuthdu.
    </p>
    <p>
      For any question, Im on 
      <a
        href="mailto:helmuthdu@gmail.com"
        target="blank"
      >
        helmuthdu@gmail.com
      </a>
    </p>
    <p>
      For any issues, any PR are welcoming
      <a
        href="https://github.com/helmuthdu/react-start"
        target="blank"
      >
         on this repository
      </a>
    </p>
  </section>
</DocumentFragment>
`;
